tools/benchmark

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+LEGOFY=$(pwd)/bin/legofy;
+PATH_IMAGES=$(pwd)/images/benchmark/;
+
+benchmark() {
+  local flags=$@;
+
+  # Default arguments
+  if [[  $flags = '' ]]; then
+    flags='--filter-quality=80 --pattern-quality=80 --quality=good'
```

Comment:
I wasn't aware we could give a number here, interesting. Where it is documented? May want to tweak `--pixel-interval` too?